command/git_push_test.go

```diff
@@ -0,0 +1,137 @@
+package command
+
+import (
+	"context"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/model/patch"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/rest/client"
+	"github.com/google/shlex"
+	"github.com/mongodb/jasper"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExecuteGitPush(t *testing.T) {
```

Comment:
It looks like these tests have some repeated content that might be better served by using a common fixture. Have you seen some of the subtests that annie has been writing recently? That might be a good place to start.

Reply:
that's cool.
I restructured the test.